molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/entity/impl/ExacAnnotator.java

```diff
@@ -0,0 +1,148 @@
+				.setDescription("African minor allele frequency derived straight from the ExAC AFR_MAF field").setLabel(EXAC_AFR_MAF_LABEL);
+
+		DefaultAttributeMetaData exac_asn_maf = new DefaultAttributeMetaData(EXAC_ASN_MAF, FieldTypeEnum.DECIMAL)
+				.setDescription("Asia minor allele frequency derived straight from the ExAC ASN_MAF field").setLabel(EXAC_ASN_MAF_LABEL);
+
+		DefaultAttributeMetaData exac_eur_maf = new DefaultAttributeMetaData(EXAC_EUR_MAF, FieldTypeEnum.DECIMAL)
+				.setDescription("European minor allele frequency derived straight from the ExAC EUR_MAF field").setLabel(EXAC_EUR_MAF_LABEL);
+
+		DefaultAttributeMetaData exac_ea_maf = new DefaultAttributeMetaData(EXAC_EA_MAF, FieldTypeEnum.DECIMAL)
+				.setDescription("European American minor allele frequency derived straight from the ExAC AE_MAF field").setLabel(EXAC_EA_MAF_LABEL);
+
+		attributes.add(exac_gmaf);
+		attributes.add(exac_afr_maf);
+		attributes.add(exac_amr_maf);
+		attributes.add(exac_asn_maf);
+		attributes.add(exac_ea_maf);
+		attributes.add(exac_eur_maf);
+		attributes.add(exac_aa_maf);
+
+		AnnotatorInfo exacInfo = AnnotatorInfo
+				.create(Status.BETA,
+						AnnotatorInfo.Type.POPULATION_REFERENCE,
+						"exac",
+						" The Exome Aggregation Consortium (ExAC) is a coalition of investigators seeking to aggregate"
+						+ " and harmonize exome sequencing data from a wide variety of large-scale sequencing projects"
+						+ ", and to make summary data available for the wider scientific community.The data set provided"
+						+ " on this website spans 60,706 unrelated individuals sequenced as part of various "
+						+ "disease-specific and population genetic studies. ",
+						attributes);
+		EntityAnnotator entityAnnotator = new AnnotatorImpl(EXAC_TABIX_RESOURCE, exacInfo, new LocusQueryCreator(),
+				new VariantResultFilter(), dataService, resources);
+
+		return new RepositoryAnnotatorImpl(entityAnnotator);
+	}
+
+	@Bean
+	Resource exacResource()
+	{
+		Resource exacTabixResource = null;
+
+		DefaultEntityMetaData repoMetaData = new DefaultEntityMetaData(EXAC_TABIX_RESOURCE);
+		repoMetaData.addAttributeMetaData(CHROM_META);
+		repoMetaData.addAttributeMetaData(POS_META);
+		repoMetaData.addAttributeMetaData(REF_META);
+		repoMetaData.addAttributeMetaData(ALT_META);
+		repoMetaData.addAttributeMetaData(new DefaultAttributeMetaData("AC", DECIMAL));
+		repoMetaData.addAttributeMetaData(new DefaultAttributeMetaData("GMAF", DECIMAL));
+		repoMetaData.addAttributeMetaData(new DefaultAttributeMetaData("AFR_MAF", DECIMAL));
+		repoMetaData.addAttributeMetaData(new DefaultAttributeMetaData("AMR_MAF", DECIMAL));
+		repoMetaData.addAttributeMetaData(new DefaultAttributeMetaData("ASN_MAF", DECIMAL));
+		repoMetaData.addAttributeMetaData(new DefaultAttributeMetaData("EUR_MAF", DECIMAL));
+		repoMetaData.addAttributeMetaData(new DefaultAttributeMetaData("AA_MAF", DECIMAL));
+		repoMetaData.addAttributeMetaData(new DefaultAttributeMetaData("EA_MAF", DECIMAL));
+
```

Comment:
This is not what the file actually looks like. The AC,GMAF,AFR_MAF and so forth are one attribute in the tabix repo...